oclint-xcodebuild

```diff
@@ -185,6 +190,7 @@ def convert(input_file, output_file, is_excluded=None):
 def main():
     arg_parser = argparse.ArgumentParser(description='Converts xcodebuild logs or xctool-json-reporter logs to compile_commands.json')
     arg_parser.add_argument("-e", "-exclude", dest="exclusion", help="Directory exclusion pattern (regular expression)")
+    arg_parser.add_argument("-f", "--exclude-file", dest="file_exclusion", help="File exclusion pattern (regular expression)")
```

Comment:
I think that [convention](http://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html) dictates that options should be one character unless they are 'long options'.

Perhaps this could be a long option only?